### PR TITLE
fork change by Oscar

### PR DESCRIPTION
**Pull request by oscardilley**

### CODE REVIEW

**Comment by oscar-uob** [comment]
Can yoyu add some more stuff

**Comment by oscardilley** [comment]
I think I have requested a pull request now with a new file added

**Comment by MaksLatz** [comment]
This is a nice new file

**Comment by oscar-uob** [comment]
I agree with Max

@MaksLatz

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@ runsim.py
 Oscar Benjamin
 March 2021
 
+EDITED BY OSCAR DILLEY
```
Comment (oscar-uob) [inline README.md:5]: Please do not use all caps

**Comment by MaksLatz** [comment]
### **_this is a bold and italic title_**

**Pull request merged**